guides/GETTING_STARTED.md

```diff
@@ -25,11 +25,27 @@ We're working on some guides to outline our team culture, and some of our develo
 
 We've made a few attempts at simplifying these steps, but haven't converged on one way of doing things yet. The alternatives are detailed in our [developer setup](DEV_SETUP.md) guide. 
 
-# Usage guides
+### Usage guides
 1. (Optional) [Enable feedback form](https://github.com/ONSdigital/dp-frontend-dataset-controller#feedback-service)
 2.  [Publishing system guide](https://github.com/ONSdigital/florence/blob/develop/USAGE.md)
 3. [CMD import guide](#cmd-import-steps)
 
+# Set up for infrastructure & CI work
+
+:warning: All steps in this section link to private repositories. If the links don't work for you, revisit the steps in [setting up your mac](#set-up-your-mac-for-development-work) to ensure you're a member of our Github organization.
+
+These guides may contain some steps which have already been completed - there is no need to do anything twice, just skip to the next step of the guide.
+
+### Infrastructure
+All infrastructure configuration is managed in [dp-setup](https://github.com/ONSdigital/dp-setup) and includes the following pieces of set up:
+1. Install the [prerequisites for terraform](https://github.com/ONSdigital/dp-setup/blob/develop/terraform/README.md#prerequisites)
+2. Install the [prerequisites for ansible](https://github.com/ONSdigital/dp-setup/blob/develop/ansible/README.md#prerequisites)
+
+### Continuous Integration (CI)
+All CI configuration is managed in [dp-ci](https://github.com/ONSdigital/dp-ci) and includes the following pieces of set up:
+1. Check this [list of ansible prerequisites](https://github.com/ONSdigital/dp-ci/tree/master/ansible#prerequisites) and complete any steps not already performed
```

Comment:
I know it's in a different sub-section but this is exactly the same step as in the Infrastructure step 2 above. It could be a tiny bit confusing on initial reading. Consider revising to state something like "unless done in above step" or maybe even just using the exact same wording as the step above so it is obvious it's the same thing and doesn't need to be done twice?

Reply:
Its a weird one, it actually links to a different guide. Basically there is a list of 'how to install ansible' in both dp-setup and dp-ci, and the first ~3-5 steps are the same, and then there's a step thats essentially 'install a relevant python library to this context' and 'make sure the right keys are set up' that have subtle differences - which is what I was trying to describe here.

I wonder if perhaps we should have some centralized core guide on installing ansible, and then just those specific library steps in the dp-ci and dp-setup as needed 🤔

Reply:
now that python2 is deprecated/gone, we can (almost certainly) remove the need for pyenv

Reply:
The only reason not to centralize these steps I can think of is if there's a slight security factor of making public what versions we're using of various tooling. But we already have public lists attesting to what tools we use so this seems fairly inconsequential to me 🤔

Reply:
depends where you centralise to... dp-ci could defer to dp-setup as the main doc for (e.g.) ansible

then it's not public, it's centralised

I don't think readers care which repo it's in, and I'd prefer it to be in dp-setup because that's where most use of ansible is

Reply:
Updated this PR and added:
https://github.com/ONSdigital/dp-setup/pull/985
https://github.com/ONSdigital/dp-ci/pull/829

Will share in the review channel as well